editor/src/core/tailwind/tailwind.ts

```diff
@@ -94,8 +94,7 @@ function getTailwindConfig(
 ): Either<any, Configuration> {
   if (tailwindFile != null && isTextFile(tailwindFile)) {
     try {
-      const requireResult = requireFn('/', TailwindConfigPath)
-      const rawConfig = importDefault(requireResult)
+      const rawConfig = requireFn('/', TailwindConfigPath)
```

Comment:
Is there a non-awful way of testing the whole function?

Reply:
We can export these functions for the sake of testing, but really we need to write an end to end test (#1729), because the issue was more with the fact the `requireFn` was the one created in the canvas.